### PR TITLE
[Admin][Order] Implement and cover history view

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A
|-----------------|-----
| Branch?         | bootstrap-admin-panel
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

Screenshot 2024-05-01 at 12 00 48

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/Header/TitleBlock/actions.html.twig
Comment (Rafikooo) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/Header/TitleBlock/actions.html.twig:1]: image

It's just a design concern. I'm okay with the `action` hook. However, currently we only have one option, and if another one is needed in the future, there is plenty of space aside to add some new buttons.

File: src/Sylius/Bundle/AdminBundle/Twig/Component/Order/AddressHistoryComponent.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+use Sylius\Component\Addressing\Model\AddressLogEntry;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+#[AsLiveComponent(
```
Comment (Rafikooo) [inline src/Sylius/Bundle/AdminBundle/Twig/Component/Order/AddressHistoryComponent.php]: Actually, is it necessary to have this component live? It's just a read-only view without any interaction. Unless it's needed for `#[ExposeInTemplate]` to work.

**Review by Rafikooo — COMMENTED** [review]
image

Minor visual concerns:
1. `Updated` text on the label is barely readable.
2. In my screen date has split into two rows in some entries.
3. Some changed multi-word field labels haven't got spacing, for instance: `Provincename`.
4. Spacing line between changed items - last (or single one) one has also a line below which looks not nice.

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/Header/TitleBlock/actions.html.twig
Comment (NoResponseMate) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/Content/Header/TitleBlock/actions.html.twig:1]: Keeping as is. It's in line w/ other places.

**Comment by NoResponseMate** [comment]
@Rafikooo updated 👋 

Screenshot 2024-05-02 at 15 32 29

**Review by Rafikooo — APPROVED** [review]

**Pull request merged**